conda/recipes/ucx-py/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
+{% set py_version = environ.get('CONDA_PY', '38') %}
+{% set cuda_version = '.'.join(environ.get('CUDA', '11.2').split('.')[:2]) %}
+
+package:
+  name: ucx-py
+  version: {{ version }}
+
+source:
+  git_url: ../../..
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  string: py{{ py_version }}_g{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  script_env:
+    - VERSION_SUFFIX
+    - CC
+    - CXX
+    - CUDAHOSTCXX
+  script:
+    - {{ PYTHON }} -m pip install . -vv
```

Comment:
Do we want `-vv` here? Nothing against it, just checking if this is really what we want.

Reply:
Yeah this is something we've found is useful [in conda-forge]( https://github.com/conda-forge/staged-recipes/blob/037c3face2061a55efab6f819e89736b0cd229b4/recipes/example/meta.yaml#L38 ) for debugging. Without `-vv` we don't really get any insight into what is built or how. `-vvv` ends up being too verbose. So `-vv` is kind of the sweet spot for providing relevant information